debian/nfd.postinst

```diff
@@ -0,0 +1,21 @@
+#!/bin/sh
+set -e
+
+if [ "$1" = configure ]; then
+  # Add a user for all NDN related files and processes
+  adduser \
+      --system \
+      --quiet \
+      --home /var/lib/ndn \
+      --disabled-password \
+      --group \
+      ndn
```

Comment:
From https://wiki.debian.org/AccountHandlingInMaintainerScripts:
> It should normally not be necessary to cross-check with getent whether an account already exists since adduser --system generally does the right thing